src/services/item/plugins/published/test/index.test.ts

```diff
@@ -185,6 +185,54 @@ describe('Item Published', () => {
     });
   });
 
+  describe('GET /collections/recent', () => {
+    describe('Signed Out', () => {
+      let member;
+      let collections: Item[];
+      let categories;
+
+      beforeEach(async () => {
+        ({ app } = await build({ member: null }));
+        member = await saveMember(BOB);
+        collections = await saveCollections(member);
+        categories = await saveCategories();
+
+        // add category to a non-published item
+        const { item } = await saveItemAndMembership({ member });
+        await ItemCategoryRepository.save({ item, category: categories[0] });
+      });
+
+      it('Get 2 most recent collections', async () => {
+        const res = await app.inject({
+          method: HttpMethod.GET,
+          url: `${ITEMS_ROUTE_PREFIX}/collections/recent`,
+          query: { limit: 2 },
+        });
+        expect(res.statusCode).toBe(StatusCodes.OK);
+
+        const result = collections.sort(({ createdAt: a }, { createdAt: b }) => (a > b ? -1 : 1));
```

Comment:
I'm partial to `collections.sort((a, b) => b.createdAt - a.createdAt);` for sorting usually, but it's personal :)